test/expectations/folding_ranges/call_chained.exp.json

```diff
@@ -1,5 +1,25 @@
 {
   "result": [
+    {
```

Comment:
We had complicated logic to prevent all of these nodes from `0` to something. However, VS Code just ignores all of them and considers only the widest range to apply.

It's definitely not worth keeping all of the complex logic we had since it's already taken care by the client.

Reply:
I wonder if other editors behave the same...

Reply:
I'm not sure, but also can't think of an alternative behaviour for editors. What would they do in the case of conflicting ranges?